package.json

```diff
@@ -30,6 +30,7 @@
     "test:procfile": "node test/test-procfile.js",
     "test:eslint": "eslint --config ./.eslintrc.json \"cypress/integration/*.js\" \"network-api/networkapi/{,!(frontend)/**/}*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
     "test:scss": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" --syntax scss",
+    "test:scss:styleguide": "stylelint \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"!source/sass/**/_variables.scss\" --syntax scss --config .stylelintrc-styleguide.json",
```

Comment:
We want this task to only lint SCSS files outside of `_variables.scss`

Reply:
`:styleguide` is pretty good, it leaves the door open for other things like "only allowing font families we want" etc.